test/games/prime/patcher_file_lib/test_pickup_exporter.py

```diff
@@ -119,7 +117,12 @@ def test_calculate_hud_text(order: tuple[str, str], generic_pickup_category, def
 
 @pytest.mark.parametrize("model_style", PickupModelStyle)
 def test_create_pickup_list(
-    model_style: PickupModelStyle, empty_patches, generic_pickup_category, blank_resource_db, default_generator_params
+    model_style: PickupModelStyle,
+    empty_patches,
+    generic_pickup_category,
+    ammo_pickup_category,
```

Comment:
type hints

Reply:
gonna leave these be for now and look into making mypy work with fixtures better later